Formula/gopls.rb

```diff
@@ -0,0 +1,20 @@
+class Gopls < Formula
+  desc "Language server for the Go language"
+  homepage "https://github.com/golang/tools/tree/master/gopls"
+  url "https://github.com/golang/tools/archive/gopls/v0.5.3.tar.gz"
+  sha256 "f3a1c142f42c7e9cff62ec9a1a6e41e59305c7519735839ecedccf17597ae00e"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+  depends_on "jsonlint" => :test
+
+  def install
+    Dir.chdir("gopls") do
+      system "go", "build", *std_go_args
+    end
+  end
+
+  test do
+    system "#{bin}/gopls api-json | jsonlint"
```

Comment:
Can we check the output here?

Reply:
Its a really large Output... Alot of lines... Would that still be OK?

Reply:
Just check for a part of it. With `assert_match` for example.

Reply:
i took now a little bit of the json and did a `assert_match`. i hope its ok like this.

Reply:
If this command outputs a valid JSON document, you can parse it with `JSON.parse` and then check some hash fields and values.

Reply:
done that... what do you think?